File: C++/1012.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main(){
	double a,b,c, tri, cir, tra, qua, ret;
	cin >> a >> b >> c;
	tri = (a * c)/2.000;
	cir = 3.14159* pow(c,2);
	tra = (c*(a+b))/2.0;
	qua = pow(b,2);
	ret = a*b;

	cout <<"TRIANGULO: " << fixed << setprecision(3) << tri  << endl;
	cout <<"CIRCULO: " << fixed << setprecision(3) << cir  << endl;
	cout <<"TRAPEZIO: " << fixed << setprecision(3) << tra << endl;
	cout <<"QUADRADO: " << fixed << setprecision(3) << qua << endl;
	cout <<"RETANGULO: "  << fixed << setprecision(3)<< ret << endl;

    return 0;
}

File: Java/1012.java
import java.util.Scanner;

public class 1012 {
    public static void main(String[] args) {
        Scanner read = new Scanner(System.in);
        double a, b, c;
        a = read.nextDouble();
        b = read.nextDouble();
        c = read.nextDouble();
        System.out.printf("TRIANGULO: %.3f\n", (a*c)/2.0);
        System.out.printf("CIRCULO: %.3f\n", 3.14159*c*c);
        System.out.printf("TRAPEZIO: %.3f\n", ((a+b)/2.0)*c);
        System.out.printf("QUADRADO: %.3f\n", b*b);
        System.out.printf("RETANGULO: %.3f\n", a*b);
    }
}
